### PR TITLE
Add GLFW support to //ui/gl

### PR DESCRIPTION
**Pull request by abarth**
This patch is the first step towards getting the GLFW configuration up and
running.

### CODE REVIEW

**Comment by abarth** [comment]
@collinjackson 

Hat tip to @lyceel

File: ui/gl/gl_context_glfw.cc [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+}
+
+GLContextGlfw::~GLContextGlfw() {
+  Destroy();
+}
+
+#if !defined(OS_ANDROID)
+bool GLContextGlfw::GetTotalGpuMemory(size_t* bytes) {
+  DCHECK(bytes);
+  *bytes = 0;
```
Comment (collinjackson) [inline ui/gl/gl_context_glfw.cc]: This seems like maybe it's unimplemented, not a big deal but might leave a comment if so.

File: ui/gl/gl_surface_glfw.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ui/gl/gl_surface.h"
+
+namespace gfx {
```
Comment (collinjackson) [inline ui/gl/gl_surface_glfw.h]: add include guard like 
#ifndef UI_GL_GL_SURFACE_GLFW_H_
#define UI_GL_GL_SURFACE_GLFW_H_

File: ui/gl/gl_context_glfw.cc (hunk 1 above)
Comment (abarth) [inline ui/gl/gl_context_glfw.cc]: Done.

File: ui/gl/gl_surface_glfw.h (hunk 2 above)
Comment (abarth) [inline ui/gl/gl_surface_glfw.h]: Done.

**Comment by collinjackson** [comment]
lgtm

**Pull request merged**